### PR TITLE
Bug 1803192: *: duplicate shared configmap in openshift-config-managed namespace

### PR DESCRIPTION
**Pull request by simonpasquier**

### CODE REVIEW

File: pkg/manifests/manifests.go [hunk 1]
```diff
@@ -1043,11 +1048,19 @@ func (f *Factory) ThanosQuerierRoute() (*routev1.Route, error) {
 	return r, nil
 }
 
+func (f *Factory) SharingConfigDeprecated(promHost, amHost, grafanaHost, thanosHost *url.URL) *v1.ConfigMap {
```
Comment (s-urbaniak) [inline pkg/manifests/manifests.go]: let's add a unit test in `manifests_test.go`

**Comment by s-urbaniak** [comment]
one mini-nit, else lgtm, thank you!

File: pkg/manifests/manifests.go (hunk 1 above)
Comment (simonpasquier) [reply pkg/manifests/manifests.go]: done

**Comment by s-urbaniak** [comment]
side note: i believe we need to set up a role binding against the `openshift-config-managed` namespace in https://github.com/openshift/cluster-monitoring-operator/blob/master/manifests/0000_50_cluster_monitoring_operator_02-role.yaml.

it might be that e2e fails because we have no permission to create a configmap in that namespace.

**Comment by simonpasquier** [comment]
> it might be that e2e fails because we have no permission to create a configmap in that namespace.

I wasn't sure. If e2e tests fail because of this, I'll add the required permission for sure.

**Pull request merged**